bool increment
---
BrickCommand brick_command
bool complete
